oauth/login_service.go

```diff
@@ -121,6 +122,11 @@ func (s *LoginService) LoginCallback(
 		if err != nil {
 			return nil, err
 		}
+	case "entraid":
```

Comment:
nog niet mergen, dan verander ik het straks ook nog naar microsoft

Reply:
dat hoeft voor mij niet hoor, er komt binnenkort toch sowieso een refactor aan van dit systeem in apollo